Resources/Storage/StorageElement.py

```diff
@@ -400,28 +400,38 @@ def getLocalPlugins( self ):
       return S_ERROR( self.errorReason )
     return S_OK( self.localPlugins )
 
-  def getStorageParameters( self, plugin ):
+  def getStorageParameters( self, plugin = None, protocol = None ):
```

Comment:
No, for srm for example. You don't know whether the plugin that will take care is SRM2 or GFAL2_SRM2